code/modules/jobs/job_types/adventurer/types/combat/rare/assassin.dm

```diff
@@ -154,7 +154,7 @@
 				cloak = /obj/item/clothing/cloak/raincloak/furcloak
 				head = /obj/item/clothing/head/fancyhat
 				backr = /obj/item/gun/ballistic/revolver/grenadelauncher/bow
-				beltr = /obj/item/weapon/sword/rapier/dec
+				beltr = /obj/item/weapon/sword/iron
```

Comment:
This is meant to match the lord

Reply:
yes that ones meant to be the lord one . if you make it normal it kills the whole point of the disguise